internal/keepers/service.go

```diff
@@ -178,37 +181,31 @@ func (s *onDemandUpkeepService) processLatestHead(ctx context.Context) {
 
 	// Get only the active upkeeps from the contract. This should not include
 	// any cancelled upkeeps.
-	keys, err := s.registry.GetActiveUpkeepKeys(ctx, "0")
+	blockKey, keys, err := s.registry.GetActiveUpkeepKeys(ctx, "0")
```

Comment:
nit. rename to getLatestActiveUpkeepKeys?

Reply:
We need to keep the block key param because the implementation in pkg/chain uses it